src/ucp/wireup/wireup_cm.c

```diff
@@ -275,9 +275,51 @@ ucp_cm_ep_sa_data_pack(ucp_ep_h ep, ucp_wireup_sockaddr_data_base_t *sa_data,
     }
 }
 
+static ucs_status_t
+ucp_cm_ep_priv_space_is_enough(ucp_ep_h ep, const ucp_tl_bitmap_t *tl_bitmap,
+                               ucs_log_level_t log_level,
+                               unsigned ep_init_flags,
+                               ucp_object_version_t sa_data_version)
+{
+    ucp_worker_h worker   = ep->worker;
+    ucp_context_h context = worker->context;
+    unsigned pack_flags   = ucp_cm_address_pack_flags(worker);
+    size_t ucp_addr_size, sa_data_length;
+    ucp_rsc_index_t cm_idx;
+    ucs_status_t status;
+
+    if (ucp_ep_get_cm_wireup_ep(ep)->flags & UCP_WIREUP_EP_FLAG_SEND_CLIENT_ID) {
+        pack_flags |= UCP_ADDRESS_PACK_FLAG_CLIENT_ID;
+    }
+
+    if (ep_init_flags & UCP_EP_INIT_CREATE_AM_LANE_ONLY) {
+        pack_flags |= UCP_ADDRESS_PACK_FLAG_AM_ONLY;
+    }
```

Comment:
can we unite common code with ucp_cm_ep_priv_data_pack?

Reply:
Yes. I'll do it in next PR update

Reply:
Done